benches/graphemes.rs

```diff
@@ -1,13 +1,8 @@
 mod common;
 
 use common::{LARGE, MEDIUM, SMALL, TINY};
```

Comment:
They were already imported here. The `common` module defines its constants in the same way, `include_str!`ing the same files.